server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Free & Fair Colorado RLA System
+ * 
+ * @title ColoradoRLA
+ * @created Aug 12, 2017
+ * @copyright 2017 Free & Fair
+ * @license GNU General Public License 3.0
+ * @creator Joe Kiniry <kiniry@freeandfair.us>
+ * @description A system to assist in conducting statewide
+ * risk-limiting audits.
+ */
+
+package us.freeandfair.corla.endpoint;
+
+import static us.freeandfair.corla.asm.ASMEvent.DoSDashboardEvent.INDICATE_FULL_HAND_COUNT_CONTEST_EVENT;
+
+import spark.Request;
+import spark.Response;
+
+import us.freeandfair.corla.asm.ASMEvent;
+
+/**
+ * The endpoint for indicating that a contest must be hand-counted.
+ * 
+ * @author Joe Kiniry <kiniry@freeandfair.us>
+ * @version 0.0.1
+ */
+@SuppressWarnings("PMD.AtLeastOneConstructor")
+public class IndicateHandCount extends AbstractDoSDashboardEndpoint {
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public EndpointType endpointType() {
+    return EndpointType.POST;
+  }
+  
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String endpointName() {
+    return "/hand-count";
+  }
+
+  /**
+   * @return STATE authorization is necessary for this endpoint.
+   */
+  public AuthorizationType requiredAuthorization() {
+    return AuthorizationType.STATE;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  protected ASMEvent endpointEvent() {
+    return INDICATE_FULL_HAND_COUNT_CONTEST_EVENT;
+  }
+  
+  /**
+   * Indicate that a contest must be hand-counted.
+   * 
+   * @param the_request The request.
+   * @param the_response The response.
+   */
+  @Override
+  public synchronized String endpoint(final Request the_request, 
+                                      final Response the_response) {
+    ok(the_response, "Contests selected");
+    // try {
```

Comment:
This is really going to be exactly the same code, unless we take the trouble to come up with another data format. What will likely change is some amount of data validation where we'll check to see that the ContestToAudit objects have the right audit type in them for the particular endpoint.